vars/artifactSetVersion.groovy

```diff
@@ -33,6 +33,13 @@ enum GitPushMode {NONE, HTTPS, SSH}
      * @possibleValues `true`, `false`
      */
     'commitVersion',
+    /**
+      * Prints some more information for troubleshooting. May reveal security relevant information. Usage is recommanded for troubleshooting only. Productive usage
+      * is not recommanded.
+     * @possibleValues `true`, `false`
+      */
+    'debug',
```

Comment:
I guess we should use `verbose` as in other steps as well

Reply:
why not, but other PR, I suppose ...

Reply:
I mean instead of `debug`.
We should prevent adding yet another parameter with basically the same notion and `verbose` is already pretty wide spread across other steps ...

Reply:
aaaahhhh !! understood.

Reply:
adjusted